Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -86,6 +86,15 @@ public extension ReceiptRenderer {
                     table { background-color:#F5F5F5; width:100%; color: #707070 }
                     table td:last-child { width: 30%; text-align: right; }
                     table tr:last-child { color: #000000; }
+                    .card-icon {
+                       width: 24px;
+                       height: 17px;
+                       vertical-align: bottom;
```

Comment:
The alignment isn't working perfectly, but since I'm tweaking the font sizes further in an upcoming PR, we can leave it like this for now.